lib/backy/cli.rb

```diff
@@ -0,0 +1,75 @@
+require "thor"
+
+module Backy
+  class CLI < Thor
+    desc "backup", "Perform a backup operation using the current configuration settings"
+    def backup
+      result = Backy::PgDump.new.call
+      puts "Backup created: #{result}"
+    end
+
+    desc "push", "Push the local dump to a remote storage location"
+    def push
+      file_name = Backy::List.new.select(&:local?).last
```

Comment:
Getting an error from this line. Maybe you have to add call?
```suggestion
      file_name = Backy::List.new.call.select(&:local?).last
```

Reply:
Probably, I haven't been able to test much yet.